index.html

```diff
@@ -5,27 +5,146 @@
+                </li>
+                <li class="hamburguer">
+                <a class="hamburguer-link" href="about.html"><h3>About</h3></a>
+                </li>
+            </ul>
+        </div>
 
-        </section>
+        <!--Section flex container-->
+        <section class="flex-container">
+            <!--Hero-->
+            <div class="hero-container">
+                <div class="hero-title">
+                    <h2>"Hello! Sharing Courses"</h2>
+                    <h1>Jose Delphin System Advisor Courses 2023</h1>
+                </div>
+                <div class="sub-theme">
+                    <p>The best web course believing in the value of openness and sharing, creating a positive 
+                        change with people from over 80 countries is taking place in January, in Microverse.</p>
+                </div>
+                <div class="date-venue">
+                    <h3>2023.01.09(MON) ~ 13(FRI)</h3>
+                    <p>@ Microverse, HTTP-CSS-JavaScript and more
+                    </p>
+                </div>
+            </div>
+    
+            <!--Program-->
+            <div id="program" class="program">
+                <h2>Main Program</h2>
+                <hr>
+                <ul class="program-details">
+                    <li class="program-card">
+                        <img src="./assets/images/program_icon_01.png" alt="lecture icon">
+                        <h2>Lecture</h2>
+                        <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+                    </li>
+                    <li class="program-card">
+                        <img src="./assets/images/program_icon_02.png" alt="cc eshibition icon">
+                        <h2>CC Exhibition</h2>
+                        <p>Appreciate various creations applying CC license of artists, organized from Microverse</p>
+                    </li>
+                    <li class="program-card">
+                        <img src="./assets/images/program_icon_03.png" alt="forum icon">
+                        <h2>Forum</h2>
+                        <p>Have the time to share your thoughts and opinions with experts for each topic.</p>
+                    </li>
+                    <li class="program-card">
+                        <img src="./assets/images/program_icon_04.png" alt="workshop icon">
+                        <h2>Workshop</h2>
+                        <p>Try creating your own work using open source license rather than just watching at it.</p>
+                    </li>
+                    <li class="program-card">
+                        <img src="./assets/images/program_icon_05.png" alt="cc ignite icon">
+                        <h2>CC Ignite</h2>
+                        <p>get opportunities to network with JDP affiliates around the world, also after the summit.</p>
+                    </li>
```

Comment:
- [ ] Great job for all your great design I really appreciate your hard work I just have one note to say in your main section as I can see here the text is not clear I can see it hardly because of the backfround I recommend to increase it to let the user be able to see the text and that's important for accessiblity reasons so kindly increase it to match the given design as well ♻🔥🚀🙏🏻💯

- [ ] I think it would be great if you align the elements the icon, heading and the text to be in the center to match the design as much as possible 🙏🏻💯💯✨✨

- Your design
![image](https://user-images.githubusercontent.com/58553711/211970566-6a4ffd88-713b-48f2-847d-411946397543.png)


- Given design
![image](https://user-images.githubusercontent.com/58553711/211970622-104fbe28-3dd2-4ef3-8953-af2f6992b839.png)

Reply:
Thank you very much for the review, i am working on it.